tasks/base.py

```diff
@@ -7,9 +7,9 @@ class BaseTest(object):
     def download_mspec(self):
         if not os.path.isdir("../mspec"):
             run("cd .. && git clone --depth=100 --quiet https://github.com/ruby/mspec")
-            run("cd ../mspec && git checkout 4bcdee51406abc739ebe2a6be81ec5a93447125e")
+            run("cd ../mspec && git checkout b363f08c6ddfbda5cf868afe9c3cc10122a0b9ab")
 
     def download_rubyspec(self):
-        if not os.path.isdir("../rubyspec"):
+        if not os.path.isdir("../spec"):
             run("cd .. && git clone --depth=100  --quiet https://github.com/ruby/spec")
-            run("cd ../spec && git checkout bdfc985bb78015614292b281cf6d2d88d4685354")
+            run("cd ../spec && git checkout 6c578cd49cb59f858d2724a687c02000e9fdaae2")
```

Comment:
The `tim/features` branch looks nice, is there a plan to merge it into master at some point?